Sources/BraintreeCore/RepeatingTimer.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+final class RepeatingTimer {
```

Comment:
```suggestion
final class BTRepeatingTimer {
```

Reply:
Yeh, we can just hard code it to 15 in the `RepeatingTimer` class. BTAnalyticsService doesn't need to change the interval